app/controllers/new_gestionnaire/dossiers_controller.rb

```diff
@@ -208,5 +208,10 @@ def mark_avis_as_read
       dossier.notifications.avis.mark_as_read
       current_gestionnaire.mark_tab_as_seen(dossier, :avis)
     end
+
+    def mark_annotations_privees_as_read
```

Comment:
Pinaillage : est-ce que le nommage en `_as_read` n'est pas hérité de l'ancien code. Si oui :
- nommer ces méthodes `*_as_seen` ?
- rajouter un `# FIXME: delete once the old code is gone` avant la première ligne de ces méthodes ?

Reply:
=> #1084